src/store/inputSlice.js

```diff
@@ -53,7 +53,8 @@ const initialState = {
     }
   },
   inverseKinematics: {
-    enabled: false
+    enabled: false,
+    lastSentArmIKState: false
```

Comment:
Lets make this a nullable boolean, and init it to `null`, which represents that we havent sent an IK update request yet